src/appleseed.studio/python/module.cpp

```diff
@@ -135,4 +136,6 @@ BOOST_PYTHON_MODULE(_appleseedstudio)
 
     bpy::def("main_window", main_window_as_pylong);
     bpy::def("create_dock_widget", create_dock_widget, bpy::args("dock_name"));
+
+    boost::python::def("get_studio_dir", get_executable_directory);
```

Comment:
Yes, that's one option I was suggesting, but I don't like that it will point to bin/Ship or bin/Debug when running studio from the sandbox.
If we return the top level studio dir instead, then maketx will be found always in result/bin and we could access other resources inside studio in an independent way.